### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Timi2424**
[DEMO LINK](https://Timi2424.github.io/react_movies-list-filter/)

### CODE REVIEW

**Comment by Timi2424** [comment]
I didn't need apply preventDefault, because we don't sumbit a form in this example, my bad

**Review by szymanski-marcin — CHANGES_REQUESTED** [review]
Can't check your demo link. Try npm run deploy and re-request review after that.

File: src/App.tsx [hunk 1]
```diff
@@ -1,31 +1,45 @@
...
+  const [query, setQuery] = useState('');
+
+  const handleSubmit: FormEventHandler = (event) => {
+    event.preventDefault();
+  };
+
+  const visibleMovies = moviesFromServer.filter(
+    item => item.title.toLowerCase().includes(query.toLowerCase())
+    || item.description.toLowerCase().includes(query.toLowerCase()),
+  );
```
Comment (mvjl000) [inline src/App.tsx:16]: You could move it out of the component and just invoke it in the jsx

File: src/App.tsx [hunk 2]
```diff
@@ -1,31 +1,45 @@
...
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleSubmit: FormEventHandler = (event) => {
+    event.preventDefault();
+  };
```
Comment (mvjl000) [inline src/App.tsx:11]: No need for this handler and form either. Input will do just fine

File: src/App.tsx [hunk 3]
```diff
@@ -1,31 +1,45 @@
...
-                placeholder="Type search word"
-              />
-            </div>
+              <div className="control">
+                <input
+                  type="text"
+                  id="search-query"
+                  className="input"
+                  placeholder="Type search word"
+                  onChange={(event) => setQuery(event.target.value.trim())}
```
Comment (mvjl000) [inline src/App.tsx:35]: Create a handler function for this onChange

**Review by mvjl000 — CHANGES_REQUESTED** [review]

**Review by mbruhler — APPROVED** [review]

**Pull request closed without merging**